index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>portfolio</title>
+</head>
+<body>
+  <header class="navigation_bar">
+    <li class="logo">My Logo</li>
+    <li class="nav">
+      <img src="./image/Vector.svg" alt="">
+      <img src="./image/Vector.svg" alt="">
+      <img src="./image/Vector.svg" alt="">
+    </li>
```

Comment:
- [ ] Hello @godybuoy, you have made a great effort, but I would suggest you use an anchor tag `a` here instead of a `li` tag, and it should be out of the `nav` tag. This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 👏 👏 👏 

- [ ] Kindly wrap the hamburger `li` tag inside a `nav` tag. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

Hint on what am talking about: 👇 

```HTML
<header>
     <a href="#">My Logo</a>
     <nav>
         <ul class="nav-header">
            <li><img src="./image/Vector.svg" alt=""></li>
            <li><img src="./image/Vector.svg" alt=""></li>
            <li><img src="./image/Vector.svg" alt=""></li>
         </ul>
     </nav>
</header>
```